tracer/test/Datadog.Trace.Tests/Sampling/RuleBasedSamplerTests.cs

```diff
@@ -75,9 +79,10 @@ public void Keep_Half_Rule()
             sampler.RegisterRule(new CustomSamplingRule(0.5f, "Allow_nothing", ".*", ".*"));
```

Comment:
While you're here maybe we could fix this name?

```suggestion
            sampler.RegisterRule(new CustomSamplingRule(0.5f, "Allow_half", ".*", ".*"));
```

Reply:
Thanks. Fixed in 253f45a3f965a5a6493bade26bb7927c9b9c1de7.